src/view/components/common/dropdown-component/dropdown-component.html

```diff
@@ -0,0 +1,11 @@
+<button md-icon-button (click)="onToggle()" [disabled]="disabled" *ngIf="icon">
```

Comment:
Here we are missing the title and the  ```<article>``` to wrap the components, please check the others.